tests/integrations/huggingface/cassettes/test_huggingface_inference_client/test_huggingface_translation_async.yaml

```
@@ -1,38 +1,404 @@
 interactions:
+    body: ''
+    headers:
+      accept:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 8 but found 6

(indentation)